aws_terraform/unsupported_lambda_runtime.rego

```diff
@@ -1,9 +1,8 @@
 package main
 
 valid_runtimes = {
+	"nodejs18.x",
 	"nodejs16.x",
-	"nodejs14.x",
```

Comment:
Small request to add nodejs14.x back in since it's still supported for a bit longer.  I'll set a reminder to get it taken out.